AdyenActions/Components/3DS2/ThreeDS2Component.swift

```diff
@@ -49,24 +54,28 @@ public final class ThreeDS2Component: ActionComponent {
         
         /// The configuration for Delegated Authentication.
         public struct DelegatedAuthentication {
-            /// The localized reason string show to the user during registration.
-            public let localizedRegistrationReason: String
+            // The relying party identifier that is used for PassKeys.
+            // See: https://developer.apple.com/documentation/xcode/supporting-associated-domains
+            // See: https://developer.apple.com/documentation/authenticationservices/public-private_key_authentication/supporting_passkeys
+            public let relyingPartyIdentifier: String
 
-            /// The localized reason string show to the user during authentication.
-            public let localizedAuthenticationReason: String
+            /// The configuration for Delegated Authentication Component style
+            public let delegatedAuthenticationComponentStyle: DelegatedAuthenticationComponentStyle
 
-            /// The Apple registered development team identifier.
-            public let appleTeamIdentifier: String
+            /// The localization parameters, leave it nil to use the default parameters.
+            public let localizationParameters: LocalizationParameters?
 
             /// Initializes a new instance.
             ///
-            /// - Parameter localizedRegistrationReason: The localized reason string show to the user while registration flow.
-            /// - Parameter localizedAuthenticationReason: The localized reason string show to the user while authentication flow.
-            /// - Parameter appleTeamIdentifier: The Apple registered development team identifier.
-            public init(localizedRegistrationReason: String, localizedAuthenticationReason: String, appleTeamIdentifier: String) {
-                self.localizedRegistrationReason = localizedRegistrationReason
-                self.localizedAuthenticationReason = localizedAuthenticationReason
-                self.appleTeamIdentifier = appleTeamIdentifier
+            /// - Parameter relyingPartyIdentifier: The relying party identifier that is used for PassKeys
+            /// - Parameter delegatedAuthenticationComponentStyle: The delegated authentication component style.
+            /// - Parameter localizationParameters: The localization parameters, leave it nil to use the default parameters.
+            public init(relyingPartyIdentifier: String,
```

Comment:
This struct is deep in the hierarchy but nonetheless the init is public and it's changed. If (albeit very unlikely) anyone is using the init directly, it's gonna be a breaking change for them.

Reply:
Yes in the strictest sense the public api has to change. 
Though we know that Delegated authentication is not used by anyone as yet, which will change in sometime.